### PR TITLE
Svg logo, sticky header, new socialmedia icons

### PR DESCRIPTION
**Pull request by ericfennis**
- Svg logo added to the header
- The header is now sticky and show up when scroll to top
- Socialmedia icons are svg now
- New header html added to al pages

[_Aspoz_] Fixes: Issue #2 #3

### CODE REVIEW

File: app/contact.html [hunk 1]
```diff
@@ -37,9 +37,11 @@
     	</div>
     </div>
     <div class="layout">
-      <div class="header">
+       <div class="header">
```
Comment (sndrgrdn) [inline app/contact.html]: Remove whitespace, there is a setting a sublime that will do this for you

File: app/events-borrel-28sept.html [hunk 2]
```diff
@@ -36,9 +36,11 @@
     	</div>
     </div>
     <div class="layout">
-      <div class="header">
+       <div class="header">
```
Comment (sndrgrdn) [inline app/events-borrel-28sept.html]: whitespace

File: app/events.html [hunk 3]
```diff
@@ -36,9 +36,11 @@
     	</div>
     </div>
     <div class="layout">
-      <div class="header">
+       <div class="header">
```
Comment (sndrgrdn) [inline app/events.html]: whitespace

File: app/index.html [hunk 4]
```diff
@@ -38,9 +38,11 @@
     	</div>
     </div>
     <div class="layout">
-      <div class="header">
+      <div class="header">
```
Comment (sndrgrdn) [inline app/index.html]: whitespace

File: app/lidworden.html [hunk 5]
```diff
@@ -35,22 +35,24 @@
     	</div>
     </div>
     <div class="layout">
-      <div class="header">
+       <div class="header">
```
Comment (sndrgrdn) [inline app/lidworden.html]: whitespace

File: app/nieuws.html [hunk 6]
```diff
@@ -36,9 +36,11 @@
     	</div>
     </div>
     <div class="layout">
-      <div class="header">
+       <div class="header">
```
Comment (sndrgrdn) [inline app/nieuws.html]: whitespace

File: app/styles/objects/_header.sass [hunk 7]
```diff
@@ -1,17 +1,34 @@
...
+  -o-transition: top 300ms ease-out 0ms
+  -webkit-transition: top 300ms ease-out
+  -webkit-transition-delay: 0ms
+  transition: top 300ms ease-out 0ms
+
+  &.sticky 
+    z-index: 10
+
+  &.sticky.nav-up
+      top: -96px
```
Comment (sndrgrdn) [inline app/styles/objects/_header.sass]: change to 6rem

File: app/styles/objects/layouts/_base.sass [hunk 8]
```diff
@@ -41,3 +41,5 @@ body
 
 .hide
   display: none
+.layout 
+  margin-top: 96px
```
Comment (sndrgrdn) [inline app/styles/objects/layouts/_base.sass:45]: change to 6rem

File: app/vacature.html [hunk 9]
```diff
@@ -36,9 +36,11 @@
     	</div>
     </div>
     <div class="layout">
-      <div class="header">
+       <div class="header">
```
Comment (sndrgrdn) [inline app/vacature.html]: whitespace

File: app/vacaturebank.html [hunk 10]
```diff
@@ -36,9 +36,11 @@
     	</div>
     </div>
     <div class="layout">
-      <div class="header">
+       <div class="header">
```
Comment (sndrgrdn) [inline app/vacaturebank.html]: whitespace

File: app/wiezijnwij.html [hunk 11]
```diff
@@ -36,9 +36,11 @@
     	</div>
     </div>
     <div class="layout">
-      <div class="header">
+       <div class="header">
```
Comment (sndrgrdn) [inline app/wiezijnwij.html]: whitespace

File: npm-debug.log [hunk 12]
```diff
@@ -0,0 +1,26 @@
+0 info it worked if it ends with ok
```
Comment (sndrgrdn) [inline npm-debug.log:1]: what is this?
if this keeps appearing on windows perhaps putting this file in gitignore will be better

File: app/scripts/main.js [hunk 13]
```diff
@@ -17,3 +17,52 @@ var closeMenu = document.querySelector('.menu-close');
...
+        // Scroll Up
+        if(st + $(window).height() < $(document).height()) {
+            $('.header.sticky').removeClass('nav-up').addClass('nav-down');
+        }
+    }
+    
+    lastScrollTop = st;
+}
+
+}		);
```
Comment (sndrgrdn) [inline app/scripts/main.js]: add a new line to every file, this is needed for better git commits
There is also an option in sublime for adding a newline add the end of a file on save

File: app/scripts/main.js [hunk 14]
```diff
@@ -17,3 +17,52 @@ var closeMenu = document.querySelector('.menu-close');
 myMenu.addEventListener('transitionend', OnTransitionEnd, false);
 oppMenu.addEventListener('click', toggleClassMenu, false);
 closeMenu.addEventListener('click', toggleClassMenu, false);
+
+$(function() {
```
Comment (sndrgrdn) [inline app/scripts/main.js:21]: inconsistent indenting

**Review by sndrgrdn — CHANGES_REQUESTED** [review]
Overall the navigation works great, it only feels a little slow to show when scrolling up.
The Facebook social icon color seems a little off.

**Comment by ericfennis** [comment]
Thanks for the feedback!
Can you take a look at the last commit?

**Review by sndrgrdn — APPROVED** [review]

**Pull request merged**